### PR TITLE
Jan2021

### PR DESCRIPTION
**Pull request by andyofsmeg**

### CODE REVIEW

File: content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+In 2020, we continued to develop the [*riskmetric*](https://pharmar.github.io/riskmetric/) package and began to align development with the Risk Assessment Shiny app (see below).  At the time of writing the package includes 12 metric assessments.  
+
+We provided a workshop to guide users to implement a risk-based approach to qualify R packages at the R/Pharma conference (https://github.com/pharmaR/rpharma2020). This provided the team with additional ideas for metrics.  We are aiming to release the first version of riskmetric package to CRAN in early 2021.
+
+## Risk Assessment Application
+
+At the end summer of 2020, Fission Labs finished building the [first iteration](/blog/2020/08/05/2020-08-05-risk-assessment-application/) of the application. Since then, the application has gone through a series of updates and upgrades:
+
+- Restructured the backend to address issues like code duplication and inefficient queries to the DB
```
Comment (elong0527) [inline content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd:29]:
```suggestion
- Restructured the backend to address issues like code duplication and inefficient queries to the database
```

File: content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Testing
+
+In addition to the plans for riskmetric and the Risk Assessment App, we recently launched a new subteam with the focus of producing a white paper on testing.  And we will release an accompanying suite of tests, embededded within a re-usable qualification framework built upon testthat.  And if you read the [recent blog post](/blog/2020/09/21/2020-09-21-status-update-sept-2020) from Mark Padgham you'll note that we are maintaining close links with the ROpenSci testing initiative.
+
+## R Consortium Developments
+
+The R Validation Hub is now an R Consortium working group.  Since the start of 2020, two further working groups have been initiatied with the help of the R Consortium:
+
+* **R Tables for Regulatory Submission (RTRS):** Develop standards for creating tables that meet the requirements of FDA submission documents
```
Comment (elong0527) [inline content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd:51]:
```suggestion
* [**R Tables for Regulatory Submission (RTRS):**](https://github.com/RConsortium/rtrs-wg) Develop standards for creating tables that meet the requirements of FDA submission documents
```

File: content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+* **Submissions:** Focus on IT and platform challenges that must be addressed in order to make “all R” regulatory submissions.
```
Comment (elong0527) [inline content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd:52]:
```suggestion
* [**Submissions:**](https://github.com/RConsortium/submissions-wg) Focus on IT and platform challenges that must be addressed in order to make “all R” regulatory submissions.
```

File: content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+categories:
+  - news
+tags:
+  - riskmetric
+  - risk assessment
+  - white paper
+banner: 'img/banners/news.png'
+---
+
+2020 was a busy year for the R Validation Hub.  We released our [white paper](/white-paper/) describing our current thinking on a risk based approach to using R for regulatory work.  We started to support the implementation of the white paper with tools such as *riskmetric* and our risk assessment application. And we started a new sub-team with the aim of producing a follow-up white paper on testing.  Throughout,  we have continued to share and gain feedback on our proposed approach, presenting at [User!](https://youtu.be/WUVUjdqifJ8); running a workshop at [R/Pharma](https://pharmar.github.io/rpharma2020/); and speaking at an [EU Programming Heads meeting](/presentations/eu_prog_heads.pdf) in June.
```
Comment (elong0527) [inline content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd:15]:
```suggestion
2020 was a busy year for the R Validation Hub.  We released our [white paper](/white-paper/) describing our current thinking on a risk based approach to using R for regulatory work.  We started to support the implementation of the white paper with tools such as [*riskmetric*](https://github.com/pharmaR/riskmetric) and our [risk assessment application](https://github.com/pharmaR/risk_assessment). And we started a new sub-team with the aim of producing a follow-up white paper on testing.  Throughout,  we have continued to share and gain feedback on our proposed approach, presenting at [User!](https://youtu.be/WUVUjdqifJ8); running a workshop at [R/Pharma](https://pharmar.github.io/rpharma2020/); and speaking at an [EU Programming Heads meeting](/presentations/eu_prog_heads.pdf) in June.
```

**Review by elong0527 — APPROVED** [review]
Thanks Andy. Looks good to me. I just add few hyperlinks.

File: content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd (hunk 4 above)
Comment (jmanitz) [reply content/blog/2021-01-13-status-update-a-summary-of-2020.Rmd:15]: - We "started" a lot of things, suggest to rephrase: "And we initiated a new sub-team with the aim of producing a follow-up white paper on testing."
- Suggest to correct a auto-correct typo: "UseR!"

**Review by jmanitz — APPROVED** [review]
looks good to me, just some minor suggestions

**Pull request merged**